examples/charlcd_keypad_simpletest.py

```diff
@@ -0,0 +1,42 @@
+"""Simple test for keypad on I2C RGB character LCD Shield or Pi Plate kits"""
+import time
+import board
+import busio
+import adafruit_character_lcd.character_lcd_rgb_i2c as character_lcd
+
+# Modify this if you have a different sized Character LCD
+lcd_columns = 16
+lcd_rows = 2
+
+# Initialise I2C bus.
+i2c = busio.I2C(board.SCL, board.SDA)
+
+# Initialise the LCD class
+lcd = character_lcd.Character_LCD_RGB_I2C(i2c, lcd_columns, lcd_rows)
+
+lcd.clear()
+lcd.color = [100, 0, 0]
+while True:
+    if lcd.left_button:
+        print("Left!")
+        lcd.message = "Left!"
+
+    if lcd.up_button:
+        print("Up!")
+        lcd.message = "Up!"
+
+    if lcd.down_button:
+        print("Down!")
+        lcd.message = "Down!"
+
+    if lcd.right_button:
+        print("Right!")
+        lcd.message = "Right!"
+
+    if lcd.select_button:
+        print("Select!")
+        lcd.message = "Select!"
+
+    else:
```

Comment:
this else only matches the select button's if - y ou want elifs instead of if's for all the buttons.

Reply:
The code ran successfully when I tested it. I updated it and tested it successfully again.

Reply:
kk do you grok why you want elifs instead of if's? z good thing to know!

Reply:
Yep! It came out of testing shorter code snippets while iterating on the lib update and it stayed into the final. I was simply pointing out that I had tested the code. :)

Reply:
good! more testing tends to reveal many mysteries